src/main/resources/schema/ans/0.5.9/traits/trait_social.json

```diff
@@ -11,7 +11,8 @@
         "type": "string"
       },
       "url": {
-        "type": "string"
+        "type": "string",
+        "format": "uri"
```

Comment:
"url" is unfortunately something of a construct here -- I think we need to drop it for now for backwards compatibility